ext/nnstreamer/tensor_filter/tensor_filter_flare.cc

```diff
@@ -0,0 +1,377 @@
+          backend = SFlareApi::BackendType::GPU;
+          backend_type = "GPU";
+        } else if (g_ascii_strcasecmp (option.get ()[1], "cpu") == 0) {
+          backend = SFlareApi::BackendType::CPU;
+          backend_type = "CPU";
+        } else {
+          throw std::invalid_argument (
+              "Invalid backend value: " + std::string (option.get ()[1]));
+        }
+      } else if (g_ascii_strcasecmp (option.get ()[0], "output_size") == 0) {
+        output_size
+            = static_cast<size_t> (g_ascii_strtoull (option.get ()[1], NULL, 10));
+      } else if (g_ascii_strcasecmp (option.get ()[0], "model_type") == 0) {
+        if (g_ascii_strcasecmp (option.get ()[1], "1B") == 0) {
+          config.llm_model = SFlareApi::GaussFModelType::GAUSS1B;
+          model_type = "GAUSS1B";
+        } else if (g_ascii_strcasecmp (option.get ()[1], "3B") == 0) {
+          config.llm_model = SFlareApi::GaussFModelType::GAUSS3B;
+          model_type = "GAUSS3B";
+        } else {
+          throw std::invalid_argument (
+              "Invalid model_type value: " + std::string (option.get ()[1]));
+        }
+      } else if (g_ascii_strcasecmp (option.get ()[0], "data_type") == 0) {
+        if (g_ascii_strcasecmp (option.get ()[1], "W4A32") == 0) {
+          config.data_type = SFlareApi::ModelDataType::DTYPE_W4A32;
+          data_type = "DTYPE_W4A32";
+        } else if (g_ascii_strcasecmp (option.get ()[1], "W8A32") == 0) {
+          config.data_type = SFlareApi::ModelDataType::DTYPE_W8A32;
+          data_type = "DTYPE_W8A32";
+        } else if (g_ascii_strcasecmp (option.get ()[1], "W16A32") == 0) {
+          config.data_type = SFlareApi::ModelDataType::DTYPE_W16A32;
+          data_type = "DTYPE_W16A32";
+        } else if (g_ascii_strcasecmp (option.get ()[1], "W32A32") == 0) {
+          config.data_type = SFlareApi::ModelDataType::DTYPE_W32A32;
+          data_type = "DTYPE_W32A32";
+        } else {
+          throw std::invalid_argument (
+              "Invalid data_type value: " + std::string (option.get ()[1]));
+        }
+      } else {
+        throw std::invalid_argument (
+            "Unknown custom property " + std::string (option.get ()[0]));
+      }
+    }
+  }
+
+  std::cout << config.tokenizer_path << ":" << backend_type << ":" << output_size
+            << ":" << model_type.c_str () << ":" << data_type.c_str () << std::endl;
+
+  return;
+}
+
+/**
+ * @brief Initialize flare context with given configuration.
+ */
+void
+TensorFilterFlare::initializeFlareContext ()
+{
+  context = SFlareApi::initSFlare ();
+  if (!context) {
+    throw std::runtime_error ("Failed to initialize flare context");
+  }
+
+  SFlareApi::ErrorCode result;
+  result = context->setSFlareOptions (config);
+  if (result != SFlareApi::ErrorCode::SFLARE_SUCCESS) {
+    throw std::runtime_error ("Failed to set flare options");
+  }
+
+  result = context->loadSFlareLLMModel (backend, use_gpu);
```

Comment:
plz check 2nd param : enable_fsu
I suggest getting this value when parsing custom prop string.